docs/book/src/plugins/creating-plugins.md

```diff
@@ -64,3 +64,46 @@ Feel free to check the implementation under:
 
 - deploy-image: <https://github.com/kubernetes-sigs/kubebuilder/tree/v3.7.0/pkg/plugins/golang/deploy-image/v1alpha1>
 - grafana: <https://github.com/kubernetes-sigs/kubebuilder/tree/v3.7.0/pkg/plugins/optional/grafana/v1alpha>
+
+## Plugin scaffolding
```

Comment:
WDYT about we add a section to explain how the plugins work in a chain?
I mean, that by creating a composition we would call the init first of plugin A, then plugin B and etc.